### PR TITLE
Feat : landing page

### PR DESCRIPTION
**Pull request by anuj-kumary**
- Created folder structure
- Completed Landing Page

ScreenShot:- 

Landing-Page1
Landing-Page2

### CODE REVIEW

File: eagle-stores/src/App.js [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
+  );
+}
 
 function App() {
   return (
    <>
-   <h1>EagleStores</h1>
+   <BrowserRouter>
+    <NavBar />
+    <Carousel />
```
Comment (Rohanmond) [inline eagle-stores/src/App.js]: Carousel and HomeCategory you can put inside Home component

**Review by Rohanmond — APPROVED** [review]

**Review by samyakshah3008 — APPROVED** [review]
Loved the way how you structured your folders 
Code looks very clean with proper semantic html tags 
Approving ✅

**Comment by PeeyushPrashant** [comment]
The UI looks clean , loved the way you have structured your folders.

File: eagle-stores/src/components/Carousel/Carousel.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                        <h3 className="carousel__subheading">For all your gadgets</h3>
+                        <a href="./Pages/Product List/product-list.html" className="btn btn__default">Shop Now</a>
+                    </div>
+                </div>
+            </section>
+        </>
+    )
+}
+
+export default Carousel
```
Comment (varnita619) [inline eagle-stores/src/components/Carousel/Carousel.jsx]: you can add an extra line at the end of each file

File: eagle-stores/public/index.html [hunk 3]
```diff
@@ -14,6 +14,14 @@
...
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
+    <link rel="stylesheet" href="https://eagleui.netlify.app/css/component.css">
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/mockman-js@latest/dist/style.css"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css"
+    integrity="sha512-PgQMlq+nqFLV4ylk1gwUOgm6CtIIXkKwaIHp/PAIWHzig/lKZSEGKEysh0TCVbHJXCLN7WetD8TFecIky75ZfQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (varnita619) [inline eagle-stores/public/index.html:30]: you can also add comments like which link is for component library and which is the link for google fonts.

**Review by varnita619 — APPROVED** [review]
Rest all looks good

File: eagle-stores/public/index.html (hunk 3 above)
Comment (anuj-kumary) [reply eagle-stores/public/index.html:30]: Done.

File: eagle-stores/src/components/Carousel/Carousel.jsx (hunk 2 above)
Comment (anuj-kumary) [reply eagle-stores/src/components/Carousel/Carousel.jsx]: Done.

File: eagle-stores/src/App.js (hunk 1 above)
Comment (anuj-kumary) [reply eagle-stores/src/App.js]: Done. Create home file inside pages and put carousel and categories inside that

**Pull request merged**